app/controllers/security_group_controller.rb

```diff
@@ -14,8 +14,6 @@ def self.display_methods
     %w(instances network_ports)
   end
 
-  menu_section :net
-
```

Comment:
well if you just delete this line the menu sections will not remember the items that where last clicked